src/components/events/eventInfoCard.js

```diff
@@ -0,0 +1,248 @@
+import { media } from '../../theme/media'
+import {
+  AccessibilityIcon,
+  DateIcon,
+  GenderIcon,
+  MailIcon,
+  MapPinIcon,
+  PhoneIcon,
+  TicketIcon,
+} from '../../components/icons'
+import Button from '../../components/button'
+import PropTypes from 'prop-types'
+
+const Wrapper = styled.div`
+  background-color: ${props => props.theme.colors.indigo};
+  display: flex;
+  flex-direction: column;
+  padding: 30px 20px;
+  color: white;
+  ${media.desktop`
+    position: absolute;
+    width: 400px;
+    right: 90px;
+    top: 270px;
+    padding: 40px;
+ `};
+`
+
+const Row = styled.div`
+  display: flex;
+  margin-bottom: 14px;
+  :last-child {
+    margin-bottom: 0;
+  }
+`
+
+const IconWrapper = styled.div`
+  flex: 0 0 26px;
+  display: flex;
+  justify-content: center;
+  margin-right: 16px;
+  padding-top: 2px;
+`
+
+const Title = styled.h3`
+  margin: 0;
+  font-size: 1em;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  color: white;
+  margin-bottom: 4px;
+`
+
+const Detail = styled.p`
+  margin: 0;
+  font-size: 0.875rem;
+`
+
+const Item = ({ title, icon, detail }) => (
+  <Row>
+    <IconWrapper>{icon}</IconWrapper>
+    <div>
+      {title && <Title>{title}</Title>}
+      {detail && <Detail>{detail}</Detail>}
+    </div>
+  </Row>
+)
+
+const Hr = styled.hr`
+  border: none;
+  border-top: 1px solid ${props => props.theme.colors.eucalyptusGreen};
+  width: 100%;
+  margin: 16px 0px 32px 0px;
+`
+
+const VSpace = styled.div`
+  margin-top: 15px;
+`
+
+// I chose formats based on
+// https://www.gov.uk/service-manual/design/dates
+const dateFormat = 'D MMMM YYYY'
+
+const formatDayRange = (startTime, endTime) => {
+  if (startTime.isSame('day', endTime)) {
+    return startTime.format(dateFormat)
+  }
+
+  return `${startTime.format(dateFormat)} to ${endTime.format(dateFormat)}`
+}
+
+const timeFormat = 'h:mma'
+
+const formatTimeRange = (startTime, endTime) =>
+  `${startTime.format(timeFormat)} to ${endTime.format(timeFormat)}`
+
+const formatPrice = (eventPriceLow, eventPriceHigh) => {
+  if (eventPriceLow === 0 && eventPriceHigh === 0) {
+    return 'Free'
+  }
+  return `From £${eventPriceLow}`
+}
+
+const formatAddress = (addressLine1, addressLine2, city, postcode) => {
+  return [
+    addressLine1,
+    addressLine2,
+    [city, postcode].filter(Boolean).join(' '),
+  ]
+    .filter(Boolean)
+    .join(', ')
+}
+
+const Link = styled.a`
+  color: white;
+  text-decoration: underline;
+  :hover {
+    text-decoration: none;
+  }
+`
+
+const VENUE_DETAILS = {
+  genderNeutralToilets: 'Gender neutral toilets',
+  outdoors: 'Outdoors',
+  indoors: 'Indoors',
+}
+
+export default function EventInfoCard({
+  data: {
+    locationName,
+    addressLine1,
+    addressLine2,
+    city,
+    postcode,
+    startTime,
+    endTime,
+    eventPriceLow,
+    eventPriceHigh,
+    email,
+    phone,
+    venueDetails,
+    ticketingUrl,
+    accessibilityOptions,
+  },
+}) {
+  console.log(accessibilityOptions)
```

Comment:
Why are we logging this?

Reply:
Argh I thought I pushed a fix for this this earlier today.